lib/r.lua

```diff
@@ -458,6 +485,50 @@ function util.split_ref(ref)
   return words[1], words[2]
 end
 
+function util.make_param(modulename, module, param, polyphony, controlspecoptions, name, paramformatter, paramparamset) -- added by @andrew
```

Comment:
i'm having second thoughts about including this kind of function in the core r lib! do you have an example of this function used in practice? i couldn't find it used in the script you linked to.

Reply:
yea I'm using it over here https://github.com/AndrewShike/orgn/blob/master/orgn.lua

it's basically just a way to make params with less typing - up to you if you want to include it !